requirements-setup.txt

```diff
@@ -1,9 +1,10 @@
 # Requirements needed for the setup make target and the test target
 # outside of the individual tox environments.
 
-Django<2.0
+Django>=2.0
 pluggy==0.3.0
 py==1.4.27
 tox==2.0.1
-virtualenv==1.11.6
+tox-pyenv==1.1.0
```

Comment:
This was needed so tox could switch between python versions on mac.  Python3.6 wasn't available from homebrew so it has to be installed via `pyenv`